### PR TITLE
docs: fix the powered-by link

### LINKED ISSUE #3767: docs: fix powered-by link
# Improve Docs

## Please describe which part of docs should be improved or typo fixed

Hi, I noticed that this `powered-by.md` is missing 🤔 for this file is under the root directory.

image

## Describe the solution you'd like

1. Visit https://github.com/apache/apisix/edit/master/README.md

image

2. Use `[Powered By](powered-by.md)` instead.

### PR DESCRIPTION
**Pull request by stu01509**
### Pre-submission checklist:
Fix #3767 
Fix the powered-by link.

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: docs/es/latest/README.md [hunk 1]
```diff
@@ -323,7 +323,7 @@ Una gran variedad de compañías y organizacioes usan APISIX para investigación
 
 <img src="https://user-images.githubusercontent.com/40708551/109484046-f7c4e280-7aa5-11eb-9d71-aab90830773a.png" width="725" height="1700" />
 
-Se alienta a los usuarios a sumarse a la página [Powered By](https://github.com/apache/apisix/blob/master/powered-by.md).
+Se alienta a los usuarios a sumarse a la página [Powered By](./powered-by.md).
```
Comment (juzhiyuan) [inline docs/es/latest/README.md:326]: We could refer to the root file

**Review by juzhiyuan — COMMENTED** [review]
PoweredBy link is ok now, this PR could close IMO.

**Review by spacewander — APPROVED** [review]

**Review by juzhiyuan — CHANGES_REQUESTED** [review]
https://github.com/apache/apisix/tree/3ef09d08b5592363f88da6d83759cc82f1a6802c/docs/es/latest

This directory doesn't have powered-by.md actually 🤔 Please keep using the previous version.

**Comment by stu01509** [comment]
Seem like #3768 has been solved this problem, so close this PR.

**Pull request closed without merging**